### ISSUE TITLE
Chosen Multiple dropdown visible when all items are selected and display_selected_options=false 

### ISSUE THREAD

**Issue by cgatesman** [harvesthq/chosen#3035]
I have commented about this in #1769, but it has been closed due to activity.  This original issue was focused on IE which I have not even looked at, but this is a broader issue:

The issue I'm seeing is that when everything is selected, the `.chosen-drop` is still visible.  It may have a 0px height, but it still has its borders, and it makes the border thicker on the bottom of the chosen.  In my application I have styled out all shadows, and in this case, you can see the thicker border on the bottom caused by the visible `.chosen-drop`.

IMO, if there are no items in the `.chosen-drop`, it should not be visible at all.

Before clicking on the chosen:

![screen shot 2018-11-14 at 5 54 28 pm](https://user-images.githubusercontent.com/6276072/48518072-64b10180-e836-11e8-9832-5e282a4b631f.png)

After clicking on the chosen:

![screen shot 2018-11-14 at 5 54 36 pm](https://user-images.githubusercontent.com/6276072/48518088-6bd80f80-e836-11e8-8208-22a59323d049.png)

**Comment by tjschuck** [comment]
This seems like something you're doing specifically in your app — when you select all options in a multi select, Chosen will still expand and show the options, they'll just be disabled.  Here's an example from the Chosen demo page:

<img width="392" alt="chosen a jquery plugin by harvest to tame unwieldy select boxes 2018-11-15 11-00-16" src="https://user-images.githubusercontent.com/308724/48565072-ab017180-e8c5-11e8-90b5-cef5290f264b.png">

So I'm pretty sure this isn't an issue with Chosen natively.

That said, what browser, browser version, OS, and OS version are you using?  And what version of Chosen?  If not the latest version, does it still happen on the [latest version](https://github.com/harvesthq/chosen/releases)?

Additionally, can you link to a publicly accessible page (or a [Fiddle](http://jsfiddle.net)) that shows the issue so we can attempt to reproduce?

Closing until more info is provided, then we'll reopen.

**Comment by cgatesman** [comment]
You completely missed an important part of this.  This happens when using the `display_selected_options=false` option.

**Comment by tjschuck** [comment]
Yep, I did miss that — sorry!  With that option, I can reproduce it on the Chosen demo page (Safari Version 12.0.1 on macOS 10.14.1, but almost certainly everywhere). Reopening.

**Comment by cgatesman** [comment]
Thanks for not giving up on it!  😄
